### PR TITLE
Add docs are starting with the archetype

### PR DESCRIPTION
**Pull request by ibacher**

### CODE REVIEW

File: docs/custom-implementation.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    Install Git, Maven and Docker Compose
+
+HIS systems, like Ozone, often need to be heavily customized based on local needs. Ozone facilitates implementer's providing custom implementation configurations via a series of tools built on Apache Maven. The starting-point for a custom implementation is the Ozone Maven Archetype which creates a skeleton project that serves as a base for any particular customizations an implementation needs.
+
+## Getting Started
+
+To get started, use Maven's archetype tools to generate a new Ozone implementation project. The command for doing this is:
+
+```bash
+ mvn archetype:generate -DarchetypeArtifactId=ozone-distro-archetype -DarchetypeGroupId=com.ozonehis
```
Comment (kazlaw) [inline docs/custom-implementation.md:14]: The artifact id of the archetype should be `maven-archetype`

**Review by kazlaw — COMMENTED** [review]
@ibacher the `maven-archetype` doesn't seem to be found when using the `generate` command. How should I set up my archetype repositories for it to be picked up?

File: docs/custom-implementation.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+This will prompt you for several key variables for your implementation:
+
+* **distributionName:** A user-friendly name for your distribution. For example, an Ozone implementation for the country of Gruzinia might use "Gruzinia" as the distribution name.
```
Comment (kazlaw) [inline docs/custom-implementation.md:19]: I would suggest that archetype doesn't add the `Ozone` word to the `distribution name` as below. 

image

File: docs/custom-implementation.md (hunk 1 above)
Comment (ibacher) [inline docs/custom-implementation.md:14]:
```suggestion
 mvn archetype:generate -DarchetypeArtifactId=maven-archetype -DarchetypeGroupId=com.ozonehis
```

Comment (ibacher) [reply docs/custom-implementation.md]: Yeah, that got changed at some point.

**Comment by ibacher** [comment]
> How should I set up my archetype repositories for it to be picked up?

You need to have the standard OpenMRS Maven setup, so that the "archetype" server resolves to "https://mavenrepo.openmrs.org/public".

**Comment by rbuisson** [comment]
Regarding this
> You need to have the standard OpenMRS Maven setup, so that the "archetype" server resolves to "https://mavenrepo.openmrs.org/public".

@ibacher , could you add this to the doc itself, as well as a link to the OpenMRS documentation to set that up?

File: docs/custom-implementation.md (hunk 2 above)
Comment (rbuisson) [reply docs/custom-implementation.md:19]: Not always.

When distributions don't have a proper product name, yes we should put `Ozone` in front. When they do, then they should be able to not have to have `Ozone`.
KenyaHMIS distro for instance is named `KenyaHMIS` not `Ozone KenyaHMIS`.
But for Haiti, well, we don't have a good product name for it, so it's just `Ozone Haiti`

> **distributionName:** A user-friendly name for your distribution. For example, an Ozone implementation for the country of Gruzinia might use "Ozone Gruzinia" as the distribution name.

**Comment by rbuisson** [comment]
Thanks @ibacher . I've made another PR that I could easily amend, which largely reuses your instructions.
https://github.com/ozone-his/ozone-docs/pull/4
Closing this one.

**Pull request closed without merging**